### DISCUSSION TITLE
Readium SDK Progress

### DISCUSSION THREAD

**Discussion by JideGuru** [JideGuru/epub_reader#64] [Ideas]
Here we'll talk about updates on the Readium SDK integration.
https://github.com/JideGuru/epub_viewer/milestone/1

**Comment by 312654929** [comment]
Hello，How is it going? I really need it.

**Reply by JideGuru** [reply]
@312654929 Still working on it. I've been a bit busy with work but I'll try to finish it up as soon as posible

**Comment by syedecryptr** [comment]
Any plans on replacing master with this branch? for me, it seems Readium-reader is much better than Folioreader. Maybe you can have a poll on that? or do you have some plans regarding this?

**Reply by JideGuru** [reply]
Yes, I have plans of merging the radium into master but it's going to be gradual. I've been able to implement the Readium for the android part while the ios part still uses Folioreader. Not sure if I should publish that or wait till I'm done with ios which will take a lot of time because of the way the Readium library was made

**Reply by syedecryptr** [reply]
I checked for the android it's working perfectly smooth, thanks for such awesome work you made opensource.

I think if that's the case(ios readium will take time) I would suggest making a new repo for android-readium  with new screenshots for the same, my reasoning for that is keeping the code in a different branch of this repo makes it hard for many people to find it which don't do justice with such a gem. 
You can also mention in the new repo that this will be for only android.
What do you think?

**Reply by syedecryptr** [reply]
Also, I am assuming a lot of people would like to have a choice between the two readers, and based on their interests, likes they can choose the repo to use.

**Reply by JideGuru** [reply]
I think separating the repository is just going to stress people more

**Reply by mrifni** [reply]
any updates on this @JideGuru ?

**Reply by Dymxin-alt** [reply]
Hello @JideGuru any progress on this please?
